### PR TITLE
Dialog and movieform

### PR DESCRIPTION
**Pull request by bcvitanusic**

### CODE REVIEW

File: glomentorprogram/src/components/Dialog/Dialog.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import './Dialog.css';
+import { AiFillCloseCircle as CloseIcon } from 'react-icons/ai';
+
+const Dialog = ({ onClose, title, children }) => {
```
Comment (leandrocunha) [inline glomentorprogram/src/components/Dialog/Dialog.js:6]: @bcvitanusic your implementation looks great but I'm curious to understand something. You put the children in a conditional statement, so if it has children put it, if not render a close button with Header? So if the Dialog need to show the content, will not have a close button and header?

what about putting the children together with a close, and header, and if there is no children, render nothing? Does it make sense?

File: glomentorprogram/src/components/Dialog/Dialog.stories.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	() => {
+		return (
+			<>
+				<Dialog>
+					<MovieForm initialMovieInfo={{}} />
+				</Dialog>
+				<Dialog>
+					<MovieForm initialMovieInfo={initialMovie} />
+				</Dialog>
+				<Dialog title={'Are you sure you want to delete movie?'} />
```
Comment (leandrocunha) [inline glomentorprogram/src/components/Dialog/Dialog.stories.js]: @bcvitanusic This implementation will not display your title, it's related to my comment in the `Dialog.js`

File: glomentorprogram/src/components/Dialog/Dialog.stories.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+			action: 'close-clicked',
+		},
+	},
+};
+
+export const DialogMovieForm = {
+	render: () => {
+		return (
+			<Dialog>
+				{' '}
```
Comment (leandrocunha) [inline glomentorprogram/src/components/Dialog/Dialog.stories.js]: @bcvitanusic you can remove the empty space

**Review by leandrocunha — CHANGES_REQUESTED** [review]

File: glomentorprogram/src/components/Dialog/Dialog.js (hunk 1 above)
Comment (bcvitanusic) [reply glomentorprogram/src/components/Dialog/Dialog.js:6]: Yes, fixed.

File: glomentorprogram/src/components/Dialog/Dialog.stories.js [hunk 4]
```diff
@@ -14,6 +14,12 @@ const initialMovie = {
 export default {
 	title: 'Dialog',
 	component: Dialog,
+	parameters: {
+		baxkgrounds: {
```
Comment (leandrocunha) [inline glomentorprogram/src/components/Dialog/Dialog.stories.js]: @bcvitanusic the parameter `baxkgrounds` is correct, or did make a typo ?

**Review by leandrocunha — APPROVED** [review]

**Pull request merged**